bus-mapping/src/circuit_input_builder.rs

```diff
@@ -1357,6 +1665,7 @@ impl<P: JsonRpcClient> BuilderClient<P> {
             state_db,
             code_db,
             &eth_block,
+            Default::default(),
```

Comment:
I would suggest adding a TODO here, as the `BuilderClient` will need to build the list of 256 history hashes.

Reply:
Added in [`e862ca8`](https://github.com/appliedzkp/zkevm-circuits/pull/292/commits/e862ca89321a97048aed576e0dea3beea6a9fb52)